### PR TITLE
[Feature store] Add delete methods

### PR DESCRIPTION
**Pull request by urihoenig**

### CODE REVIEW

File: mlrun/feature_store/api.py [hunk 1]
```diff
@@ -582,3 +582,19 @@ def set_task_params(
         featureset.set_targets()
     featureset.save()
     return source, parameters
+
+
+def delete_feature_set(name, project=""):
```
Comment (yaronha) [inline mlrun/feature_store/api.py:587]: @urihoenig what about tag/ver?

File: mlrun/feature_store/api.py [hunk 2]
```diff
@@ -582,3 +582,19 @@ def set_task_params(
...
+    """ Delete a :py:class:`~mlrun.feature_store.FeatureSet` object from the DB.
+    :param name: Name of the object to delete
+    """
+    db = mlrun.get_run_db()
+    return db.delete_feature_set(name=name, project=project)
+
+
+def delete_feature_vector(name, project=""):
```
Comment (yaronha) [inline mlrun/feature_store/api.py:595]: same, tag/ver?

File: mlrun/feature_store/api.py (hunk 1 above)
Comment (Hedingber) [reply mlrun/feature_store/api.py:587]: @yaronha we're currently don't support deleting a specific version - only the whole feature set (with all versions)

File: mlrun/feature_store/api.py [hunk 3]
```diff
@@ -582,3 +582,19 @@ def set_task_params(
         featureset.set_targets()
     featureset.save()
     return source, parameters
+
+
+def delete_feature_set(name, project=""):
+    """ Delete a :py:class:`~mlrun.feature_store.FeatureSet` object from the DB.
+    :param name: Name of the object to delete
```
Comment (Hedingber) [inline mlrun/feature_store/api.py:589]: project param is missing in docstring

File: mlrun/feature_store/api.py [hunk 4]
```diff
@@ -582,3 +582,19 @@ def set_task_params(
...
+    """
+    db = mlrun.get_run_db()
+    return db.delete_feature_set(name=name, project=project)
+
+
+def delete_feature_vector(name, project=""):
+    """ Delete a :py:class:`~mlrun.feature_store.FeatureVector` object from the DB.
+    :param name: Name of the object to delete
```
Comment (Hedingber) [inline mlrun/feature_store/api.py:597]: same

**Review by Hedingber — CHANGES_REQUESTED** [review]

**Comment by urihoenig** [comment]
@Hedingber - Added the param description.
@yaronha  - I talked with Saar & Dina. We need a new ticket opened on Saar to implement the delete with tag/version (this will be on top of this PR so there's no problem to merge it).

**Comment by urihoenig** [comment]
@yaronha / @Hedingber  - Can you please merge?

**Pull request merged**